src/models/pageOrder.ts

```diff
@@ -37,52 +66,39 @@ class PageOrder {
   /**
    * Find all pages which match passed query object
    *
-   * @param {object} query
-   * @returns {Promise<Page[]>}
+   * @param {object} query - input query
+   * @returns {Promise<PageOrder[]>}
    */
-  static async getAll(query = {}) {
+  public static async getAll(query: object = {}): Promise<PageOrder[]> {
     const docs = await db.find(query);
 
-    return Promise.all(docs.map(doc => new PageOrder(doc)));
-  }
-
-  /**
-   * @class
-   *
-   * @param {PageOrderData} data
-   */
-  constructor(data = {}) {
-    if (data === null) {
-      data = {};
-    }
-
-    if (data._id) {
-      this._id = data._id;
+    if (docs === null || docs instanceof Error) {
```

Comment:
I don't like here and further logic handling via `docs instanceof Error`. Could we throw error in db.find and handle it via try-catch in other methods or anything else. The idea that docs can have Error type looks weird